README.md

```diff
@@ -113,6 +113,13 @@ the tests against every minor version of the specified dependencies.
 $ versioned-tests --minor tests/versioned/*.tap.js
 ```
 
+The following command will run only those test files whose names include the
+keyword *redis*:
+
+```sh
+$ versioned-tests tests/versioned/*.tap.js -P redis
+```
+
```

Comment:
Could add a quick blurb here for executing a single file. Although, it is somewhat intuitive.